python/GafferUI/ScriptWindow.py

```diff
@@ -102,6 +106,8 @@ def setLayout( self, compoundEditor ) :
 			del self.__menuContainer[1:]
 
 		if hasattr( compoundEditor, "settings" ) :
+			if currentEditScope and compoundEditor.settings().getChild( "editScope" ) :
```

Comment:
I think `"editScope" in compoundEditor.settings()` is the more idiomatic way of doing this, rather than using `getChild()`. Not a big deal, but if you feel like tidying it up, there are a bunch of other examples in this PR.

Reply:
Oh, but I think in this case the check is redundant anyway?

Reply:
I've removed the redundant checks from 9918e2802821385983e69c0e1adf50bb501d2e54 

and tidied up some of the `getChild()` stuff in 961f5cbe46bffd1942d882a6bdeacdbc6641d661, c40b77b87ac29bf3763041494e76ad1307bf66e0, and 48b99d0f2b6601e6bef00ffe2ca55de6c91ab970